critiquebrainz/frontend/views/test/test_mapping.py

```diff
@@ -78,64 +74,73 @@ def setUp(self):
             }
         }
 
-    def test_spotify_list(self):
+    @mock.patch('critiquebrainz.frontend.external.musicbrainz_db.release_group.get_release_group_by_id')
+    @mock.patch('critiquebrainz.frontend.external.spotify.get_multiple_albums')
+    @mock.patch('critiquebrainz.frontend.external.mbspotify.mappings')
+    def test_spotify_list(self, get_mappings, get_spotify_albums, get_release_groups):
```

Comment:
just to make it a bit easier to follow, can we try and keep method names the same as the method that it's mocking? (in this case, `mappings, get_multiple_albums, get_release_group_by_id`)

Reply:
in the case where we have to mock a method with the same name in 2 different packages, we could prefix it with part of the package name if needed